tests/logs/fix_cves/test_fix_cves/test_noop_on_same_cves.txt

```diff
@@ -0,0 +1,7 @@
+[    INFO] Get erratum: started
+[    INFO] Get erratum: finished
+[    INFO] Compare CVEs: started
+[    INFO] CVEs from erratum: ['CVE-123']
+[    INFO] CVEs for fix: ['CVE-123']
+[    INFO] New CVEs are same as the ones on the advisory. Nothing to do.
```

Comment:
As mentioned elsewhere - I don't think this should really be a no-op.  It should still go ahead and publish the repos.